agent/recorder.go

```diff
@@ -168,8 +169,10 @@ func (r *SpanRecorder) sendSpans() (error, bool) {
 
 		var testSpans int64
 		for _, span := range spans {
-			if isTestSpan(span) {
-				testSpans++
+			if tags, ok := span["tags"].(opentracing.Tags); ok {
```

Comment:
Which cases it gave a wrong test span count?

Reply:
Here: https://github.com/undefinedlabs/scope-go-agent/blob/fix-test-counter/agent/recorder.go#L181-L189

The problem with that counter, is that is being used here: https://github.com/undefinedlabs/scope-go-agent/blob/fix-test-counter/agent/report.go#L12

So if the that value is always zero, always return "success" while we have some KO in the backend...